cmake/kmeans.pc.in

```diff
@@ -0,0 +1,8 @@
+Name: @PROJECT_NAME@
+Description: @PROJECT_DESCRIPTION@
+Version: @libkmeans_VERSION_MAJOR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_LIBDIR@/modules 
+
+Cflags: -I@CMAKE_INSTALL_FULL_LIBDIR@/modules
+Libs: -L@CMAKE_INSTALL_FULL_LIBDIR@ -lkmeans @MPI_LIBRARIES@ -lmpi_usempif08
```

Comment:
No idea if I want to include `lmpi_usempif08`. I assume yes, but that it should be conditional on the build options. This file resolves to:

```
Name: libkmeans
Description: 
Version: 1
libdir=/Users/alexanderbuccheri/Codes/kmeans/install/lib
includedir=/Users/alexanderbuccheri/Codes/kmeans/install/lib/modules 

Cflags: -I/Users/alexanderbuccheri/Codes/kmeans/install/lib/modules
Libs: -L/Users/alexanderbuccheri/Codes/kmeans/install/lib -lkmeans  -lmpi_usempif08
```

except now I'm not getting modules installed

Reply:
[Here](https://sourcegraph.com/github.com/scylladb/seastar@998d29970b9207c4ba8a125684f883c363d1010a/-/blob/pkgconfig/seastar.pc.in) is an example of how messed up it can be to support that. If you want more inspiration:
https://sourcegraph.com/search?q=context:global+file:.*%5C.pc.*+%24%3CTARGET_PROPERTY&patternType=keyword&sm=0